### PR TITLE
fix discovered schema name with pascal case

### PR DESCRIPTION
**Pull request by agnes512**
Schema name should be in PascalCase.
Fixes Postgres failure in https://github.com/strongloop/loopback-datasource-juggler/pull/1819
## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-postgresql) 👈

- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
@slnode test please

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "juggler-v3": "file:./deps/juggler-v3",
     "juggler-v4": "file:./deps/juggler-v4",
     "lodash": "^4.17.4",
-    "loopback-datasource-juggler": "^3.0.0 || ^4.0.0",
+    "loopback-datasource-juggler": "^3.0.0 || ^4.18.1",
```
Comment (nabdelgadir) [inline package.json]: Won't it automatically use `^4.18.1` because of the `^`?

Comment (agnes512) [reply package.json]: The CI wasn't passing so I was trying everything I could do 😂

**Pull request merged**